kibbeh/src/ui/UserProfile.tsx

```diff
@@ -13,6 +16,56 @@ export const UserProfile: React.FC<UserProfileProps> = ({
   user,
   isCurrentUser,
 }) => {
+  const { t } = useTypeSafeTranslation();
+  const [badges, setBadges] = useState<Array<badge>>([]);
+  const [tags, setTags] = useState<Array<UserBadgeLgProps>>([]);
+  useEffect(() => {
```

Comment:
instead of using useEffect and useState here, just stick it in a regular badges variable

Reply:
done